projects/ui/src/Components/Teams/Details/AppsSection/TeamAppsSection.tsx

```diff
@@ -24,14 +26,11 @@ const TeamAppsSection = ({ team }: { team: Team }) => {
   const [showAddTeamAppSubSection, setShowAddTeamAppSubSection] =
     useState(false);
 
-  const {
-    paginatedDataSlice: paginatedApps,
-    onPageChange,
-    totalPages,
-  } = usePagination(apps, 5);
+  const customPaginationData = useCustomPagination(apps ?? [], [5, 10, 20], 0);
```

Comment:
nit: might be good to have the pagination count be a constant imported from somewhere (just for consistency)